ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -560,10 +488,7 @@ export default defineComponent({
           prevDraft: prevDraftId
         }
         setRegTableNewItem(newItem)
-        router.push({
-          name: RouteNames.DASHBOARD
-        })
-        emitHaveData(false)
```

Comment:
When you return to dashboard. The dashboard component emitsHaveData(false) so I don't believe we need to emit that here as well.

![image](https://github.com/bcgov/ppr/assets/77707952/3cc818e9-ec59-4a78-a727-338882b0b113)

![image](https://github.com/bcgov/ppr/assets/77707952/66c18da9-c625-438c-9cda-daede59c74fa)